attest/tpm.go

```diff
@@ -317,6 +317,58 @@ func (t *TPM) PCRs(alg HashAlg) ([]PCR, error) {
 	return t.tpm.pcrs(alg)
 }
 
+func (t *TPM) attestPCRs(aik *AIK, nonce []byte, alg HashAlg) (*Quote, []PCR, error) {
+	pcrs, err := t.PCRs(HashSHA1)
```

Comment:
did you typo using `HashSHA1` instead of `alg`?

Reply:
Just testing you :) :smile: jks

Done.